module/onboarding/Ping/src/Ping.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Ping.hpp"
+
+#include "extension/Configuration.hpp"
+
+#include "message/onboarding/Ping.hpp"
+#include "message/onboarding/Pong.hpp"
+
+int32_t pongcount = 0;
```

Comment:
is this meant to  be here? wouldn't it come from the Pong message?

Reply:
You can't use a global variable to cache your results. Only the protobuf messages can store these things